src/pages/creations/ProjectsList/index.tsx

```diff
@@ -41,48 +69,70 @@ export default function ProjectsList({ projects = [] }: ProjectListProps) {
   };
 
   return (
-    <styles.List>
+    <List>
       {projects.map((project) => (
         <ProjectContainer key={project.name}>
           <ProjectHeader onClick={(e: React.MouseEvent<Element, MouseEvent>) => handleClick(project, e.currentTarget)}>
```

Comment:
Note: Consider implementing a single open state for projects. If one project is opened, the others should automatically close : )

Reply:
https://github.com/defi-wonderland/web/pull/114/commits/8be9cbd700bd08853a5f76cdf5072d8a2082fc87